docs/codeql/writing-codeql-queries/metadata-for-codeql-queries.rst

```diff
@@ -49,7 +49,7 @@ The following properties are supported by all query files:
 |                       | | ``warning``             |                                                                                                                                                                                                                                                                                                                                                                       |
 |                       | | ``recommendation``      |                                                                                                                                                                                                                                                                                                                                                                       |
 +-----------------------+---------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``@security-severity``| ``<score>``               | Defines the level of severity, between 0.0 and 10.0, for queries with ``@tags security``. For more information about calculating ``@security-severity``, see the `GitHub changelog <https://gh.io/2021-07-19-codeql-security-severity>`__.                                                                                                                            |
+| ``@security-severity``| ``<score>``               | Defines the level of severity, between 0.0 and 10.0, for queries with ``@tags security``. For more information, see the `GitHub user documentation <https://docs.github.com/code-security/code-scanning/managing-code-scanning-alerts/about-code-scanning-alerts#about-alert-severity-and-security-severity-levels>`__.                                               |
```

Comment:
I don't think this links to the appropriate section: should the link be https://docs.github.com/en/code-security/code-scanning/managing-code-scanning-alerts/about-code-scanning-alerts#about-security-severity-levels?

Reply:
Thanks, Angela. The section title will change when I merge the related Docs PR. I'll check whether it's ready to merge yet.

Reply:
Ah, I see it! I'll approve this then as long as it's merged after the Docs site is updated 😄

Reply:
Thank you 💖

You should find that the link now works as expected 🤞

Reply:
Hm.. https://docs.github.com/en/code-security/code-scanning/managing-code-scanning-alerts/about-code-scanning-alerts#about-alert-severity-and-security-severity-levels still isn't working for me 🤔 does it need to be deployed or something?

Reply:
Oh wow, just did a refresh and it updated right after I commented 😅 working now!